owned-manifests/machine-api-controllers.yaml

```diff
@@ -70,7 +70,7 @@ spec:
           requests:
             cpu: 10m
             memory: 20Mi
-      - name: machine-healthcheck
+      - name: machine-healthcheck-controller-manager
```

Comment:
The same here. Either `machine-healthcheck-controller` or `machine-healthcheck-manager`. Not both.